src/Oci8/Query/OracleBuilder.php

```diff
@@ -247,4 +248,68 @@ public function clone()
     {
         return clone $this;
     }
+
+    /**
+     * Add a basic where clause to the query.
+     *
+     * @param  \Closure|string|array  $column
+     * @param  mixed  $operator
+     * @param  mixed  $value
+     * @param  string  $boolean
+     * @return \Illuminate\Database\Query\Builder|\Yajra\Oci8\Query\OracleBuilder
+     */
+    public function where($column, $operator = null, $value = null, $boolean = 'and')
+    {
+        if ($this->isJsonColumn($column)) {
+            [$column, $jsonKeys] = $this->getJsonColumnAndKeys($column);
+
+            return parent::where(parent::raw("json_value($column, '$.$jsonKeys')"), $operator, $value, $boolean);
+        }
+
+        return parent::where($column, $operator, $value, $boolean);
+    }
+
+    /**
+     * Add a "where not in" clause to the query.
+     *
+     * @param  string  $column
+     * @param  mixed  $values
+     * @param  string  $boolean
+     * @return \Illuminate\Database\Query\Builder|\Yajra\Oci8\Query\OracleBuilder
+     */
+    public function whereNotIn($column, $values, $boolean = 'and')
+    {
+        if ($this->isJsonColumn($column)) {
+            [$column, $jsonKeys] = $this->getJsonColumnAndKeys($column);
+
+            return parent::whereNotIn(parent::raw("json_value($column, '$.$jsonKeys')"), $values, $boolean);
```

Comment:
I think `$.$jsonKeys` should be wrapped as it may cause possible SQL injection. Or maybe use bindings if possible?

Reply:
> I think `$.$jsonKeys` should be wrapped as it may cause possible SQL injection. Or maybe use bindings if possible?

@yajra keys or column names don't need using binding. because those will set as column name in select statement.